code/game/objects/structures/stool_bed_chair_nest_sofa/pew.dm

```diff
@@ -0,0 +1,26 @@
+/obj/structure/bed/sofa/pew
+	name = "pew"
+	desc = "A long, hard wooden seat, often seen in churches and other places of worship, as well as courtrooms."
+	icon_state = "pewmiddle"
+	base_icon = "pewmiddle"
+	color = WOOD_COLOR_GENERIC
+
```

Comment:
Should probably set material on base type too like `material = /decl/material/solid/organic/wood`

Reply:
donezo